test/fixtures/tape-standalone.js

```diff
@@ -9,10 +9,50 @@ if (navigator.userAgent.includes("Chrome/")) {
   browser = "Firefox";
 }
 
+function getArgs(/* desc, options, fn */) {
+  let desc = "(anonymous)";
+  // By setting `timeout` to `0` here, we are allowing V8 to consider
+  // it a to have a signature of `{timeout: number}` instead of `{}`
+  // with a `timeout` property, which is easier on the C++ portion.
+  let options = {timeout: 0};
+  /** @type {function(any):void|Promise<void>} */
+  let fn;
+
+  for (let i = 0, arg; i < arguments.length; i++) {
+    arg = arguments[i];
+    switch (typeof arg) {
+      case "string":
+        desc = arg;
+        break;
+      case "function":
+        fn = arg;
+        break;
+      case "object":
+        Object.assign(options, arg);
+        break;
+    }
+  }
+
+  if (!options.timeout || options.timeout < DEFAULT_TIMEOUT) {
+    options.timeout = DEFAULT_TIMEOUT;
```

Comment:
This implementation means that a test cannot set a lower timeout. That seems like an unnecessary restriction.